### PR TITLE
R: add extension KODAMA

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@verdurin @boegel it's annoying that PR #6686 was merged before this.
how can we prevent this from happening in future?

**Comment by boegel** [comment]
@smoors Sorry about that, it's hard to keep up with incoming PRs, especially with summer holidays in between (there's still a bunch of stuff I need to catch up on).

I'm not sure we can easily do something about this, PRs sometimes get overlooked or cross each other.

Let me know if you need any help with resolving the conflicts.

**Comment by smoors** [comment]
@boegel maybe we could let travis check for PRs that have changes to the same files, and give a heads up?
don't know if that is possible at all, just an idea.

**Comment by boegel** [comment]
@smoors Conflicts aren't resolved yet?

I'm not sure if we can easily detect clashes between PRs via Travis, and there may be a good reason why a certain PR isn't getting merged (which wasn't the case here, obviously).

**Comment by smoors** [comment]
@boegel I could use your help resolving the conflicts. I added the missing extensions, not clear what to do next.

regarding the clashes, it would indeed still be up to the reviewer to decide which PR should be merged first (if at all).

**Comment by smoors** [comment]
seems to be ok now..

File: easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b.eb [hunk 1]
```diff
@@ -1860,6 +1860,9 @@ exts_list = [
     ('gee', '4.13-19', {
         'checksums': ['ebd3eb754b338dc8d89a35fff149af57c1be3aa2eb6162912c7cc52a8572c292'],
     }),
+    ('pracma', '2.1.4', {
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b.eb]: @smoors This extension is already included (same version), see line 1292 in this easyconfig.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2426.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/5f729b629a91613431cfab7507be628c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2062.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/64116a4c127fe23844f3d7d0b22fc910 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**